Mac_BSD_resources.py

```diff
@@ -130,8 +130,9 @@ def measure_resources():
       events = int(line_s[1])
   pipe.close()
 
-  # Get the max number of sockets
-  maxsockets = "unable to find max number of sockets"
+  # Get the max number of sockets, or use a sane default on machines where 
```

Comment:
Not really a major issue (because it's explained in resource#5), but I'm not really sure what is meant by sane default.  Readers might find this comment clearer on its own if it instead said that 512 was found to be the default value set on OS X versions ...

This PR is fine to me.